cypress/e2e/court-cases/court-case-details/ExceptionHandlerPrompt.cy.ts

```diff
@@ -34,19 +35,77 @@ describe("ExceptionHandlerPrompt", () => {
 
       cy.get("ul.moj-sub-navigation__list").contains("Offences").click()
       cy.get(".govuk-link").contains("Attempt to rape a girl aged 13 / 14 / 15 years of age - SOA 2003").click()
-      cy.get(".qualifierCodeTable").contains("A")
+      cy.get(".qualifier-code-table").contains("A")
     })
 
     it("Should display an error prompt when a HO100309 is raised", () => {
-      cy.task("insertCourtCasesWithFields", [{ orgForPoliceFilter: "01", hearingOutcome: HO100309.hearingOutcomeXml }])
+      cy.task("insertCourtCasesWithFields", [
+        { orgForPoliceFilter: "01", errorCount: 1, triggerCount: 1, hearingOutcome: HO100309.hearingOutcomeXml }
+      ])
 
       cy.login("bichard01@example.com", "password")
       cy.visit("/bichard/court-cases/0")
 
       cy.get("ul.moj-sub-navigation__list").contains("Offences").click()
       cy.get(".govuk-link").contains("Aid and abet theft").click()
-      cy.get(".qualifierCodeTable").contains("XX")
+      cy.get(".qualifier-code-table").contains("XX")
       cy.get(".error-prompt-message").contains(ErrorMessages.QualifierCode)
+      cy.get("button").contains("Next offence").click()
+      cy.get(".error-prompt-message").should("not.exist")
+      cy.get("button").contains("Next offence").click()
+      cy.get(".error-prompt-message").should("not.exist")
+    })
+
+    it("Should not display any error prompts when exceptions are marked as manually resolved", () => {
+      cy.task("insertCourtCasesWithFields", [
+        { orgForPoliceFilter: "01", errorCount: 1, triggerCount: 1, hearingOutcome: HO100309.hearingOutcomeXml }
+      ])
+
+      cy.login("bichard01@example.com", "password")
+      cy.visit("/bichard/court-cases/0")
+
+      cy.get("ul.moj-sub-navigation__list").contains("Offences").click()
+      cy.get(".govuk-link").contains("Aid and abet theft").click()
+      cy.get(".qualifier-code-table").contains("XX")
+      cy.get(".qualifier-code-table .error-prompt-message").contains(ErrorMessages.QualifierCode)
+      cy.get(".offences-table").contains("YY10XYZXX")
+      cy.get(".offences-table .error-prompt-message").contains(ErrorMessages.HO100306_error_prompt)
+      cy.get("#exceptions-tab").contains("Exceptions").click()
+      cy.get("button").contains("Mark as manually resolved").click()
+      cy.get("button").contains("Resolve").click()
+      cy.get(".qualifier-code-table .error-prompt-message").should("not.exist")
```

Comment:
I think it's worth navigating back to the offence here to make sure we're looking at the right page before checking that things don't exist. If we changed marking as manually resolved in the future to take us to some other page, this test would still pass even though it wasn't checking what we intended